docs/guides/qiskit-addons-cutting-gates.ipynb

```diff
@@ -0,0 +1,470 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8fd79758-2771-41ce-b17e-e8bfdf5b1023",
+   "metadata": {},
+   "source": [
+    "# Get started with circuit cutting using gate cuts\n",
+    "\n",
+    "This guide demonstrates two working examples of gate cuts with the `qiskit-addon-cutting` package. The first example shows how to reduce circuit depth (the number of circuit instructions) by cutting entangling gates on non-adjacent qubits that would otherwise incur a SWAP overhead when transpiled to hardware. The second example covers how to use gate cutting to reduce the circuit width (the number of qubits) by splitting a circuit into several circuits with fewer qubits.\n",
+    "\n",
+    "Both examples will use the [`efficient_su2`](/api/qiskit/qiskit.circuit.library.efficient_su2) ansatz and reconstructs the same observable."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e58752cf-2628-4c99-805b-4d82055b547a",
+   "metadata": {},
+   "source": [
+    "## Gate cutting to reduce circuit depth\n",
+    "\n",
+    "The following workflow reduces a circuit's depth by cutting distant gates, avoiding a large series of swap gates that would otherwise be introduced.\n",
```

Comment:
```suggestion
    "The following workflow reduces a circuit's depth by cutting distant gates, avoiding a large series of SWAP gates that would otherwise be introduced.\n",
```